### ISSUE TITLE
Non-standard camera orientation in 3D

### ISSUE THREAD

**Issue by EtienneDesticourt** [MathInspector/MathInspector#40]
Hi,

From the 3D software I have used there are two standard options for axes orientation:

A)
x points to the right,
y points up
z points towards the camera

B)
x points towards the camera
y points to the right
z points up

Currently the camera is oriented so that
x points towards the camera
y points up
z points to the left <- problematic for positive values of z which we interpret naturally as negative

Cheers

**Comment by MathInspector** [comment]
Good call.  I want to use (B).  Every math book I have the z-axis points up for 3d plots, and I have been trying to get the z-axis to point up for awhile, and keep getting stuck.

On this line, it does the following : 
```python
self.model = glm.rotate(self.model, glm.radians(-90.0), glm.vec3(0,1,0))
```
https://github.com/MathInspector/MathInspector/blob/master/mathinspector/plot/plot3d.py#L49

But whenever I rotate things so the z-axis points up, all the controls get messed up.  I would really like to get this fixed so z is up.  Do you have any idea's what to change to get that?  I have been taking shots at it every so often and can't get it.

**Comment by EtienneDesticourt** [comment]
```
self.camera_up = glm.vec3(0,0,1)
self.model = glm.rotate(self.model, glm.radians(90.0), glm.vec3(0,0,1))
```

Should do the trick

**Comment by MathInspector** [comment]
SWEET!  Thank youuu =)

Will add this to the upcoming PR after the new release of 0.9.4

Do you know how to fix the gimble lock?  I can't seem to look up properly =(

I think the solution is to use quaternions but I am kind of n00b at quaternions and have been too busy to fix the gimble lock issue.  lmk if you have any advice about that.  Thanks.
